src/main/java/com/desiato/puresynth/controllers/AudioController.java

```diff
@@ -28,6 +27,9 @@ public class AudioController {
 
     private final AudioService audioService;
 
+    public AudioController(AudioService audioService) {
```

Comment:
Why have you removed `@AllArgsConstructor`?

Reply:
Because it was creating an issue with AudioController constructor. It would not let me run the application. Right now I do not remember the specific error I was getting, but that solved the problem.

Reply:
I guess the problem was you also have:
```
@Value("${audio.files.directory}")
    private String audioFilesDir;
```

But storing a file on a local disk is not an option, so I think it's better to remove the `audioFilesDir` rather than bypass `@AllArgsConstructor`.